river: Add various string functions to stdlib (#4388)
grafana/agent: docs/sources/flow/reference/stdlib/upper.md

---
aliases:
- ../../configuration-language/standard-library/upper/
title: upper
---

# upper

> `upper` converts all cased letters in the given string to uppercase.

Reviewer: ```suggestion
`upper` converts all lowercase letters in a string to uppercase.
```

## Examples

> ```

Reviewer: ````suggestion
```river
````
